app/controllers/application_controller.rb

```diff
@@ -27,7 +27,7 @@ def configure_permitted_parameters
 	end
 
 	def set_global_info
-		@edition = Edition.find(2016)
+		@edition = Edition.find(2017)
```

Comment:
Sorry, should've been more specific. It's the leading alignment. It's a tab/4 spaces instead of two.